### DISCUSSION TITLE
Telegram

### DISCUSSION THREAD

**Discussion by FrancescoSaverioZuppichini** [projectdiscovery/notify#216] [Q&A] [answered]
Dear all,

Could you please provide detailed instructions in how to setup the telegram provider? What is the `telegram_api_key` and the `telegram_chat_id` and how do I get one? Do I need to create a bot? The doc only points to the telegram bot doc without really telling me what I am suppose to do

Cheers,

Fra

**Comment by Nikakz** [comment, marked as answer]
> Caros,
> 
> Você poderia fornecer instruções detalhadas sobre como configurar o provedor de telegramas? O que é o `telegram_api_key`e o `telegram_chat_id`e como faço para obter um? Preciso criar um bot? O documento apenas aponta para o documento do bot do telegram sem realmente me dizer o que devo fazer
> 
> Saúde,
> 
> fra

https://core.telegram.org/bots#3-how-do-i-create-a-bot

**Comment by minhnq1618** [comment]
i still don't know who to get the **telegram_api_key** and **telegram_chat_id** after reading the article

**Comment by GeorginaReeder** [comment]
Hi @minhnq1618 ! If you're still looking for the `telegram_api_key` and `telegram_chat_id`, these guides might be helpful:

`telegram_api_key` : [this Telegram guide](https://core.telegram.org/api/obtaining_api_id) and [here in Stack Overflow](https://stackoverflow.com/questions/43291868/where-to-find-the-telegram-api-key) - BotFather is really useful here.

`telegram_chat_id` : [this guide](https://www.alphr.com/find-chat-id-telegram/) and [this one](https://diyusthad.com/2022/03/how-to-get-your-telegram-chat-id.html) walk you through the process.

Hope that helps. If you have any other questions, just let us know!

**Reply by minhnq1618** [reply]
Thanks, i got it now
